### DISCUSSION TITLE
Discussion on whether reaction order = stoichiometric coefficients

### DISCUSSION THREAD

**Discussion by jiweiqi** [DENG-MIT/CRNN#25] [Ideas]
E.g., [Q1](https://chemistry.stackexchange.com/questions/75960/why-are-the-stoichiometric-coefficients-the-powers-in-the-rate-law)

![image](https://user-images.githubusercontent.com/8445647/112910704-f77a3000-90c1-11eb-8fe7-a8d59370a4be.png)

**Comment by yewalenikhil65** [comment]
https://chem.libretexts.org/Bookshelves/Physical_and_Theoretical_Chemistry_Textbook_Maps/Supplemental_Modules_(Physical_and_Theoretical_Chemistry)/Kinetics/03%3A_Rate_Laws/3.02%3A_Reaction_Mechanisms/3.2.01%3A_Elementary_Reactions

I find this explanation useful. The answer in stackexchange you have posted seems perfect with this explanation. 
reaction order = stoichiometric coeffficients for elementary reactions, but not so for non-elementary reactions(set of elementary steps)


another good example of multi-step(non-elementary) reactions is here https://en.wikipedia.org/wiki/Molecularity#Difference_between_molecularity_and_order_of_reaction

**Reply by jiweiqi** [reply]
Thanks, Nikhil for sharing the links. They are very useful.

Following this logic, for inferencing non-elementary reactions (often called lumped reactions), it is not necessary to assume the reaction orders are integers as well. They are not even following a power law.
